external/vcm/vcm/cubedsphere/coarsen_restarts.py

```diff
@@ -204,19 +224,112 @@ def coarsen_restarts_on_pressure(
     return coarsened
 
 
+def coarsen_restarts_via_hybrid_method(
```

Comment:
Agreed...we could go with something like `blended_method`, since I already use "blend" in other parts of this PR, and it is what Chris refers to the method as in his writeup.

Reply:
Sure that sounds good.